MandelaK/SendIT-API
# #[161709517] Create endpoint for getting one order

#### What does this PR do?
Added endpoints for getting only one order if we specify the order id. 

#### Description of Task to be completed?
We need endpoints that allow users to get information on any one order granted they know the order ID. That is what this PR seeks to complete.

#### How should this be manually tested?
Cloning this repository and installing all dependencies on requirements.txt then starting the app and accessing GET /api/v1/parcels/<int:id> will send you back information regarding only the parcel with the provided ID. You should ensure you add only valid IDs, else you get an error message.

#### Any background context you want to provide?
No.
#### What are the relevant pivotal tracker stories?
#161709517
